rules/windows/privilege_escalation_unusual_svchost_childproc_childless.toml

```diff
@@ -18,7 +18,7 @@ name = "Unusual Service Host Child Process - Childless Service"
 risk_score = 47
 rule_id = "6a8ab9cc-4023-4d17-b5df-1a3e16882ce7"
 severity = "medium"
-tags = ["Elastic", "Host", "Windows", "Threat Detection", "Defense Evasion"]
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Defense Evasion", "Privilege Escalation"]
```

Comment:
```suggestion
tags = ["Elastic", "Host", "Windows", "Threat Detection", "Privilege Escalation"]
```

Reply:
So I left the test (and automated change) to allow for more than 1 tactic, so long as it is defined in the threat mapping. Even though the norm is to just tag the primary tactic, I did not want to be too restrictive - unless we think we should limit to only a single tactic.